### PR TITLE
nc: use xtimer for gnrc_ipv6_nc_t::nbr_sol_timer

### PR DESCRIPTION
**Pull request by cgundogan**
replaces vtimer by xtimer for gnrc_ipv6_nc_t::nbr_sol_timer.

### CODE REVIEW

**Comment by authmillenon** [comment]
Given that you are using the same code snippet over and over again, maybe an inline function in ndp somewhere would be helpful. Something like `gnrc_ndb_reset_nbr_sol_timer()`.

**Comment by authmillenon** [comment]
Needs rebase.

**Comment by cgundogan** [comment]
@authmillenon addressed your comment

File: sys/include/net/gnrc/ndp/internal.h [hunk 1]
```diff
@@ -189,6 +189,24 @@ bool gnrc_ndp_internal_mtu_opt_handle(kernel_pid_t iface, uint8_t icmpv6_type,
...
+ * @brief   Resets the gnrc_ipv6_nc_t::nbr_sol_timer.
+ *
+ * @internal
+ *
+ * @param[in] nc_entry      A neighbor cache entry.
+ * @param[in] delay         The delay when the timer should fire.
+ * @param[in] type          The msg_t::type for the timer.
+ * @param[in] pid           The pid of the receiver thread of the msg_t
+ */
+void gnrc_ndp_internal_reset_nbr_sol_timer(gnrc_ipv6_nc_t *nc_entry, uint32_t delay,
```
Comment (authmillenon) [inline sys/include/net/gnrc/ndp/internal.h]: `static inline` missing.

**Comment by cgundogan** [comment]
@authmillenon addressed comment

**Comment by cgundogan** [comment]
shall I squash?

**Comment by authmillenon** [comment]
Still need to test, but ok to squash.

**Comment by cgundogan** [comment]
rebased

**Comment by authmillenon** [comment]
Still works with both native and iotlab-m3 => go.

**Pull request merged**